lesson_06/homeworks_06.py

```diff
@@ -22,12 +34,36 @@
 + напишіть цикл for що перебере і обробить всі значення списку alien_color
 """
 
+alien_color = ['green', 'yellow', 'red']
+
+for color in alien_color: 
+    if color == 'green':
+        print("You got 5 points")
+    elif  color == 'red':   
+        print("You got 15 points")
+    else:
+        print("You got 10 points")
+
+
 # task 5
 """  Начинки для піци (pizza_topping): напишіть цикл, який пропонує користувачеві ввести ряд начинок
 для піци, доки він не введе значення 'quit'. Коли вони введуть кожну начинку,
 надрукуйте повідомлення про те, що ви додасте цю начинку до їхньої піци.
 """
 
+pizza_topping =  ''
+all_pizza_toppings = []
+
+while pizza_topping != 'quit':
+    pizza_topping = input("Enter pizza topping you would like to add. If nothing else enter 'quit':")
+    if pizza_topping == 'quit':
+        break
```

Comment:
```suggestion
    if pizza_topping == 'quit':
        pass
```
нагадайте, щоб я пояснив це на уроці